src/Files.App/Keyboard/Actions/ClearSelectionAction.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Files.App.ViewModels;
+
+namespace Files.App.Keyboard.Actions
+{
+	internal class ClearSelectionAction : IKeyboardAction
```

Comment:
Instead of IKeyboardActions, we'd like to have generic command files. It's essentially the same thing but with a more generic interface.

Reply:
Perhaps this should just be a regular command so that we can use the commands in other areas besides keyboard shortcuts.

Reply:
It is an excellent idea. The word command has a specific meaning in .net. Maybe IAction, because we don't have to handle CanExecute and CanExecuteChange, but other properties. This can also be used for certain mouse shortcuts or custom buttons or custom menu entries.

Reply:
@d2dyno1